### ISSUE TITLE
error

### ISSUE THREAD

**Issue by WP1223** [WubbaLubbaDubDubDev/notpixel_bot_advanced#32]
NotPixel | 2024-11-27 14:59:21 | ERROR    | 13 - Error occurred: too many values to unpack (expected 3)

why? i try create json for session always get this

**Comment by WubbaLubbaDubDubDev** [comment]
> NotPixel | 2024-11-27 14:59:21 | ERROR    | 13 - Error occurred: too many values to unpack (expected 3)
> 
> why? i try create json for session always get this 

Oh, so someone finally managed to explain when exactly this error happens? Shocking! 🫣 Thanks to that, I figured out what the problem was. Go ahead and install the latest update!

**Comment by WubbaLubbaDubDubDev** [comment]
> cant use proxy? i tried to use proxy from scrape, data impulse, bird all 407 error
> 

I’m currently troubleshooting an issue with proxies. It seems to be a problem with SOCKS proxies again, as HTTP proxies are working fine for me.

**Comment by WubbaLubbaDubDubDev** [comment]
> > cant use proxy? i tried to use proxy from scrape, data impulse, bird all 407 error
> > 
> 
> I’m currently troubleshooting an issue with proxies. It seems to be a problem with SOCKS proxies again, as HTTP proxies are working fine for me.
> 
> 

Yes, that's it. The issue occurred due to the update of the aiohttp-socks library; it needs to be rolled back to version 0.8.4. I'll fix the requirements.txt file and release an update.

**Comment by WubbaLubbaDubDubDev** [comment]
> cant use proxy? i tried to use proxy from scrape, data impulse, bird all 407 error
> 

@WP1223 

The proxy issue has been fixed. Everything is working properly for me now, both with HTTP and SOCKS proxies, which previously had issues. I’m looking forward to your feedback to confirm that everything is working correctly on your end as well.

**Comment by WubbaLubbaDubDubDev** [comment]
@WP1223 
A funny situation happened: I forgot to do a git push after fixing the "too many values to unpack" error. As a result, the update was only on my computer. I've fixed it now. 🥴
